week08_pomdp/practice_tensorflow.ipynb

```diff
@@ -78,7 +82,6 @@
     "    return env\n",
```

Comment:
Above this line, the PyTorch version crops 15px from the top of the image by default. In this version, the default is 5. I'm not sure if that actually matters, but this is non-trivial difference between the two notebooks.

Reply:
> A few more comments.
> 
> Have you tested the updated notebook end-to-end?

Yep, I have. Well, the testing was like to check whether learning curves grow and show similar behavior to the PyTorch version, that nothing crashes with restarting and running everything again.

Reply:
> Above this line, the PyTorch version crops 15px from the top of the image by default. In this version, the default is 5. I'm not sure if that actually matters, but this is non-trivial difference between the two notebooks.

Yeah, it looks like it came from the previous TensorFlow version. I'll unify them if you want

Reply:
> I'll unify them if you want
Please do.

Reply:
Done